Hikmani Syariful Fajar

hikmanisyariful opened the pull request.

hikmanisyariful committed:
Server ecommerce cms done

details:
1. 40 test for user and product
2. CRUD product
3. Register and Login user
4. Authenthication User
5. Authorization User as admin and User as customer

hikmanisyariful committed:
add env template

hikmanisyariful committed:
remove coverage from repository

hikmanisyariful committed:
repush

isrotrip commented on test/product.test.js:
```diff
@@ -0,0 +1,655 @@
+let adminToken
+let customerToken
+let productId
+
```
testingnya udah bagus ya, mantap

isrotrip commented on models/category.js:
```diff
@@ -0,0 +1,18 @@
+  }
+
+  Category.init({
+    name: DataTypes.STRING
```
ini jangan lupa dikasih validasi juga ya

hikmanisyariful committed:
Seeding data

details:
1. added data product to seeding products
2. make seeding to category
3. make router and controller to category
4. in product find all added included category

hikmanisyariful committed:
deploy

hikmanisyariful committed:
deploy

hikmanisyariful committed:
add routing to cart customer

hikmanisyariful committed:
clear console
